boxoffice/templates/ticket_assignment_mail.html

```diff
@@ -105,25 +127,64 @@
+          {%- elif org.details.logo %}
+            <img src="{{ org.details.logo }}" class="logo"/>
+          {%- endif %}
+          <div class="event">
+            <p class="event-title">{{ item_collection.title }}</p>
+            <p class="event-date">
+            {%- if line_item.item.details.date -%}
+              <span>{{ line_item.item.details.date }}</span>
+            {%- endif %}
+            {%- if line_item.item.details.time -%}
+              &nbsp; &#124; &nbsp;<span>{{ line_item.item.details.time }}</span>
+            {%- endif %}
+            </p>
+          </div>
+        </div>
+        <div class="ticket-body">
+          <p class="type">Ticket</p>
+          <p class="content">{{ line_item.item.title }}</p>
+          <p class="type">Name</p>
+          <p class="content">{{ line_item.current_assignee.fullname }}</p>
+          {%- if line_item.current_assignee.details.company -%}
+            <p class="type">Company</p>
+            <p class="content">{{ line_item.current_assignee.details.company }}</p>
+          {%- endif %}
+          {%- if line_item.current_assignee.details.twitter -%}
+            <p class="type">Twitter</p>
+            <p class="content twitter-handle">@{{ line_item.current_assignee.details.twitter | replace("@", "") }}</p>
+          {%- endif %}
+          {%- if line_item.item.details.venue -%}
+            <p class="type">Venue</p>
+            {%- if line_item.item.details.venue_url -%}
+              <p class="content"><a href="{{line_item.item.details.venue_url}}">{{ line_item.item.details.venue }}</a></p>
+            {%- else -%}
+              <p class="content">{{ line_item.item.details.venue }}</p>
+            {%- endif %}
+          {%- endif %}
+          <hr class="seperator">
+          <p class="type">Receipt No: {{ order.invoice_no }}</p>
+          <hr class="seperator">
+          <p class="content">
+            {%- if item_collection.details.twitter -%}
+              <a href="https://twitter.com/{{ item_collection.details.twitter | replace('@', '') }}" class="twitter-handle">@{{ item_collection.details.twitter | replace("@", "") }}</a>
+            {%- endif %}
+            {%- if item_collection.details.website -%}
+              &nbsp; &#124; &nbsp;<a href="{{ item_collection.details.website }}">{{ item_collection.details.website }}</a>
+            {%- endif %}
+          </p>
+        </div>
+      </div>
+      <p><a class="edit-button" href={{"{base_url}/order/{access_token}/ticket".format(base_url=base_url, access_token=order.access_token)}} target="_blank">Edit details</a></p>
     </div>
 
-    <p><a class="button" href={{"{base_url}/order/{access_token}/ticket".format(base_url=base_url, access_token=order.access_token)}} target="_blank">Edit details</a></p>
-    <div class="terms-content">{{org.details.get('ticket_faq', '') | safe}}</div>
+    <div class="terms-content">{{ org.details.get('ticket_faq') | safe }}</div>
```

Comment:
This template has many hardcoded assumptions on the fields in the `details` JSON dictionary on various models. These fields need to be reflected in the data model itself, as there is no way to know what fields are expected without reading each line of the templates.